aten/src/ATen/cuda/CUDAGraph.cpp

```diff
@@ -122,8 +122,24 @@ void CUDAGraph::capture_begin(MempoolId_t pool/*=0*/) {
   // The user has no business launching kernels on capture_stream_ from another thread
   // while calling capture_begin. They'll have no idea if their side thread's
   // kernel will end up as part of the capture or not.
+  // Addendum: beginAllocateStreamToPool is now called before cudaStreamBeginCapture which should
```

Comment:
I think the comment above this is now pretty out of date: the caching allocator doesn't call cudaStreamGetCaptureInfo anymore, and notifyCaptureBegin doesn't exist.